plugin/main/src/kotlinx/benchmark/gradle/BenchmarksPlugin.kt

```diff
@@ -37,8 +37,8 @@ class BenchmarksPlugin : Plugin<Project> {
 
         plugins.withType(org.jetbrains.kotlin.gradle.plugin.KotlinBasePlugin::class.java) { kotlinPlugin ->
             logger.info("Detected Kotlin plugin version '${kotlinPlugin.pluginVersion}'")
-            if (!getKotlinVersion(kotlinPlugin.pluginVersion).isAtLeast(1, 9, 0)) {
-                logger.error("JetBrains Gradle Benchmarks plugin requires Kotlin version 1.9.0 or higher")
+            if (!getKotlinVersion(kotlinPlugin.pluginVersion).isAtLeast(1, 9, 20)) {
+                logger.error("JetBrains Gradle Benchmarks plugin requires Kotlin version 1.9.20 or higher")
```

Comment:
README states that the minimum required version is 1.7.20. Should it be updated as well?

Reply:
Discussed that the docs will be updated later.

Reply:
Here is the PR to update README: https://github.com/Kotlin/kotlinx-benchmark/pull/167